<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="l_img">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>15</y>
      <width>731</width>
      <height>391</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>take_photo.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="b_start">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>420</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start </string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_stop">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>420</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>420</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>fps :</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_fps">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>420</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ch_stream">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Live Stream</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="ch_write">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Video File</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWebcam_Video_Live">
    <property name="title">
     <string>Webcam Video Capture</string>
    </property>
   </widget>
   <addaction name="menuWebcam_Video_Live"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
